app/models/essay.py

```diff
@@ -0,0 +1,8 @@
+from .. import db
+
+
+class Essay(db.Model):
+	__tablename__ = "essays"
+	id = db.Column(db.Integer, primary_key=True)
+	name = db.Column(db.String, index=True)
+	student_profile_id = db.Column(db.Integer, db.ForeignKey('student_profile.id'), nullable=False)
```

Comment:
Should there be another value here for an essay link/text?

Reply:
Yep!